### ISSUE TITLE
Show Archived Emails in Stream

### ISSUE THREAD

**Issue by Kharg** [espocrm/espocrm#2607]
Currently the Stream will show Sent and Received emails if linked to the parent entity, this features is really useful, but it would be even more useful if it could show archived emails has well (provided they have a time/date), as of now linking an archived email will not show it in the stream of the parent record.

**Comment by yurikuzn** [comment]
If you make account - email relationship audited, when archiving an email related to an account, it will appear in the account stream.

**Comment by Kharg** [comment]
Yes, this is may be a good temporary workaround, but it doesn't show the email body in the stream like Sent/Received emails like you can do by adding the entity in "streamEmailWithContentEntityTypeList" config.

**Comment by Kharg** [comment]
I’ve seen the improvements in #3237

would it be possible to also show the body for archived emails as well? 

This would be a nice improvement.

I understand if it’s not in the roadmap as of now.

**Comment by yurikuzn** [comment]
These improvements are not closely related to this issue.
